src/main/java/bookface/model/person/UniquePersonList.java

```diff
@@ -134,4 +135,17 @@ private boolean personsAreUnique(List<Person> persons) {
         }
         return true;
     }
+
+
+    /**
+     * Loans to a person {@code person} a book {@code book} .
+     */
+    public void loan(Person person, Book book) {
+        CollectionUtil.requireAllNonNull(person, book);
+        //if (person.hasSameLoanedBook(person, book.getTitle())) {
+        //    throw new DuplicateBookException();
+        //}
+        int index = internalList.indexOf(person);
```

Comment:
Was this code supposed to be commented also?

On an unrelated note, I think it's best to do `loanee.addLoanedBookTitle(book.getTitle());` here instead of in Book.java, as Person itself can deal with how to add the book title